megamek/src/megamek/common/Entity.java

```diff
@@ -14202,10 +14202,10 @@ public static String getEntityMajorTypeName(long typeId) {
      */
     public static String getEntityTypeName(long typeId) {
 
-        if ((typeId & ETYPE_BIPED_MECH) == ETYPE_BIPED_MECH) {
-            return "Biped Mech";
-        } else if ((typeId & ETYPE_LAND_AIR_MECH) == ETYPE_LAND_AIR_MECH) {
+        if ((typeId & ETYPE_LAND_AIR_MECH) == ETYPE_LAND_AIR_MECH) {
```

Comment:
It doesn't look like there are any actual code changes here, can you point out what changed substantively here? If it's just random code re-arrangement, let's roll this file back.

Reply:
the unit typeId is additive.   So the order of the checks matter to get the correct name.

So a Landair Mech is a Biped Mech and a Landair Mech.
Orignial code a Landair Mech was returning as Biped Mech.
With the new code order Landair Mech returns as Landair Mech.

Reply:
Good thing we don't use this method for determining anything important, it must have been this way for quite a while. [nervous laughter]